docs/api-docs/store-logs/store-logs.mdx

```diff
@@ -69,6 +69,99 @@ Accept: application/json
 </Tab>
 </Tabs>
 
+### Filter by date created
+
+Filter logs by using the `date_created` value in the query parameter. The date created must be in [Unix Timestamp](https://www.unixtimestamp.com/) format.
+
+The following example retrieves entries by the date created:
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```http filename="Example request: Filter by date created" showLineNumbers copy
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/store/systemlogs?date_created:min=1657688400&date_created:max=1658379600
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+</Tab>
+<Tab>
+
+```json filename="Example response: Filter by date created" showLineNumbers copy
+{
+    "data": [
+        {
+            "id": 63,
+            "type": "payment",
+            "module": "Test Payment Gateway",
+            "severity": "success",
+            "summary": "Test payment for order #121 processed successfully",
+            "message": "",
+            "date_created": "2023-01-05T22:31:54+00:00"
+        },
+        {
+            "id": 64,
+            "type": "notification",
+            "module": "Email Message",
+            "severity": "success",
+            "summary": "Notification sent successfully",
+            "message": "<strong>Sent email notification to:</strong><br />jane.doe@example.com",
+            "date_created": "2023-01-05T22:31:54+00:00"
+        },
+        {
+            "id": 65,
+            "type": "notification",
+            "module": "Optimized Checkout",
+            "severity": "success",
+            "summary": "Order has been successfully finalized via Optimized checkout.",
```

Comment:
Should it be "Optimized Checkout" or "optimized checkout" ?

Reply:
The responses all say Optimized Checkout and I left it as is. I agree that it should be lowercase but since this is what the user will see when they get the response, I kept it.